x-pack/libbeat/management/api/configuration.go

```diff
@@ -88,3 +96,8 @@ func ConfigBlocksEqual(a, b ConfigBlocks) bool {
 
 	return reflect.DeepEqual(a, b)
 }
+
+// IsConfigurationNotFound returns true if the configuration was not found.
+func IsConfigurationNotFound(err error) bool {
+	return err == errConfigurationNotFound
+}
```

Comment:
As a get more familiar with the code I might add _Transient_ and _Permanent_, which would result in use the cache version or invalid cache until the remote situation is fixed. I didn't to introduce more logic then necessary here.